docs/install/macos.rst

```diff
@@ -13,7 +13,7 @@ Install
 
 .. note:: The Kolibri Mac app is not signed, which means that you must tell MacOS to allow it to run. Read `more information about opening unsigned apps <https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac>`_ from Apple.
 
-#. Download the `.dmg installer <https://learningequality.org/download/>`_ for Kolibri **version 0.13**.
+#. Download the `.dmg installer <https://learningequality.org/r/kolibri-mac-latest>`_ for Kolibri **version 0.13**.
```

Comment:
updating to a direct link due to  https://github.com/learningequality/learningequality.org/pull/486